manifests/xylem.pp

```diff
@@ -7,6 +7,7 @@
   $gluster_mounts,
   $gluster_replica = undef,
   $gluster_stripe  = undef,
+  $redis_host      = '127.0.0.1',
```

Comment:
Would prefer not to default to an explicit IP address. Can use `$::ipaddress_lo` rather.